bin/cake.bat

```diff
@@ -17,7 +17,6 @@
 
 :: In order for this script to work as intended, the cake\console\ folder must be in your PATH
```

Comment:
I think we should remove this comment in both major versions as it is highly misleading and wrong.